applications/zeeve_larch_phase_2.md

```diff
@@ -0,0 +1,209 @@
+- Created a relay chain on substrate with some customisations done at the core to accommodate the tokenomics and custom reward mechanism [AXIA Blockchain Portal](https://axscan.axiacoin.network/#/explorer)
+- Published a [blog post](https://www.zeeve.io/blog/how-to-use-substrate-to-create-parachains-in-polkadot/) about the usage and implementation of parachains
+- Spoke with David Hawig, Richard Casey and Gautam Dhameja from the Parity team regarding the development of Zeeve and the Web3 Grant application
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 6 months
+- **Full-Time Equivalent (FTE):**  4
+- **Total Costs:** 24,000 USD
+
+### Milestone 1 — Monitoring Support for K8 deployments & Global config
+
+- **Estimated duration:** 45 Days
+- **FTE:**  4
+- **Costs:** 7,500 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | GPLv3 |
+| **0b.** | Documentation | We will provide the **tutorial** to that explains how a user can create a network configuration and start the network |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 1. | Monitoring for k8 based deployment | We will add Monitoring support to kubernetes based deployment |
+| 2. | Zombienet global parameters support | Add All possible Global parameters in template creation corresponding to the supported Zombienet version [settings](https://paritytech.github.io/zombienet/network-definition-spec.html#settings) |
+| 3. | In App docs update | Update in app docs to reflect the features added in the current milestone |
+| 4. | Update Zombienet version | Update supported Zombienet version to latest stable version at the time of release |
+
+### Milestone 2 — Export & Import of config and Parachain & Relaychain config 
+
+- **Estimated duration:** 40 Days
+- **FTE:**  4
+- **Costs:** 8,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | GPLv3 |
+| **0b.** | Documentation | We will provide the **tutorial** to that explains how a user can create a network configuration and start the network |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 1. | Zombienet config import & export | We will add Zombienet config import & export support of the template, for example config import as shown in the **Upload Config to template (Mock)** and **Config Download (Mock)**. The while uploading the config it will validate the config and prefill the template UI to perform further modification through UI on existing config. |
```

Comment:
>  The while uploading the config it will validate the config and prefill the template UI to perform further modification through UI on existing config.

Please double-check this sentence, there seems to be something missing.

Also, please specify the file type or standard that the config file will adhere to.

Reply:
fixed and provided the supported formats